toolkit/resources/assets/efi/grub/grub.cfg

```diff
@@ -1,6 +1,11 @@
 search -n -u {{.BootUUID}} -s
+search.fs_uuid {{.BootUUID}} root
```

Comment:
These look redundant.

Reply:
We looked into it and it seems these two are functionally identical. -s implicitly sets 'root' even though that is not documented anywhere...